PSE Landing Page Banner

ZaydenGit opened the pull request:
<img width="1888" height="688" alt="image" src="https://github.com/user-attachments/assets/51a4c3f6-d511-4193-a202-143e969dee10" />

Fixes #11 
Needs a fix to properly scale background. It should be slightly more zoomed in, I just wanted to push this tonight.
The directory for the asset used in the background may need to be changed.
The positioning of elements may be _slighty_ off but I tried to follow the figma.

swedenagu commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,27 @@
+      <Image
+        src={PSEBg}
+        alt="PSE Background"
+        className="center absolute inset-0 -top-[20px] -z-10 flex w-full scale-120 object-cover"
```
the banner should be positioned closer to the top of the page. remember to leave room for the navbar

swedenagu commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,27 @@
+const PSEBanner = () => {
+  return (
+    <div
+      className={`relative flex h-[459px] w-full flex-col items-center justify-center overflow-clip`}
```
see if you can make the height relative to screen size for responsiveness but where it'll look about the same as this pixel size

swedenagu commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,27 @@
+        Pi Sigma Epsilon
+      </p>
+      ``
+      <p className="font-pse-gfs-didot z-10 -mt-8 text-center text-[40px] text-white">
```
responsify text size (with breakpoints)

ZaydenGit commented:
<img width="1883" height="666" alt="image" src="https://github.com/user-attachments/assets/6a9f654b-e7fe-4a6e-9280-a2abe7c105d9" />
Fixed scaling of each element and changed font for subtitle

ZaydenGit commented:
Added breakpoints to properly scale each element for sm, md, and lg displays.
<img width="1888" height="676" alt="image" src="https://github.com/user-attachments/assets/85d88fe2-4050-4da0-a8ee-f71ba6c02b93" />

swedenagu commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PSEBg from "@/public/landing/pse_bg.png";
```
this should be a webp

Howardwheeler commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,26 @@
+      <p className="font-pse-crimson-text scale-50 text-center text-[125px] whitespace-nowrap text-white transition-transform duration-300 ease-out sm:scale-60 md:scale-75 lg:scale-100">
+        Pi Sigma Epsilon
+      </p>
+      <p className="font-pse-maitree z-10 -mt-18 scale-50 text-center text-[40px] whitespace-nowrap text-white transition-transform duration-300 ease-out sm:-mt-16 sm:scale-60 md:-mt-10 md:scale-75 lg:-mt-6 lg:scale-100">
```
just a quick fix but just to keep as a habit for the rest of the project, try using tailwind's pre-set font sizes instead of pixels. It doesn't have to be perfectly accurate but helps make the code more clear. (https://v2.tailwindcss.com/docs/font-size) PS* Its also recommended to avoid using [--px] if possible because it makes things static but i dont think its an issue atm so the rest looks good to me :)

brandontran1220 commented on src/app/page.tsx:
```diff
@@ -1,7 +1,10 @@
+      <PSEBanner />
       <JoinUsButton
         link="https://docs.google.com/forms/d/e/1FAIpQLSco0HkuBMDgfyNt2JmmyuYf9OYI5aGxlGIkl4B9omNPD74zWw/viewform"
         text="JOIN US!"
```
Copy this info for when you call the button in your component. Then, delete it from here after.

brandontran1220 commented on src/app/page.tsx:
```diff
@@ -1,7 +1,10 @@
+import PSEBanner from "@/components/landing/PSEBanner";
 import JoinUsButton from "@/components/landing/JoinUsButton";
```
```suggestion

```

brandontran1220 reviewed (changes requested):
Hey Zayden, it looks really good! I was waiting for the button to be made. Can you import the button and call it within your component. Refer to the figma on spacing/design. Just copy the properties listed in page.tsx.

brandontran1220 commented on src/app/page.tsx:
```diff
@@ -1,16 +1,9 @@
-"use client";
-
-import Principles from "@/components/landing/Principles";
```
Looks like you deleted principles. Add this component back to page.tsx

brandontran1220 commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,34 @@
+      <p className="font-pse-maitree z-10 -mt-18 scale-50 text-center text-[40px] whitespace-nowrap text-white transition-transform duration-300 ease-out sm:-mt-16 sm:scale-60 md:-mt-10 md:scale-75 lg:-mt-6 lg:scale-100">
+        University of California, Riverside Chapter
+      </p>
+      <div className="relative z-10 mt-0 scale-80 transition-transform duration-300 ease-out hover:bg-red-500 sm:mt-2 sm:scale-90 md:mt-8 md:scale-100 lg:mt-10 lg:scale-110">
```
```suggestion
      <div className="relative z-10 mt-0 scale-80 transition-transform duration-300 ease-out sm:mt-2 sm:scale-90 md:mt-8 md:scale-100 lg:mt-10 lg:scale-110">
```

brandontran1220 commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,34 @@
+      </p>
+      <p className="font-pse-maitree z-10 -mt-18 scale-50 text-center text-[40px] whitespace-nowrap text-white transition-transform duration-300 ease-out sm:-mt-16 sm:scale-60 md:-mt-10 md:scale-75 lg:-mt-6 lg:scale-100">
+        University of California, Riverside Chapter
+      </p>
```
Try using the text sizes available already. Try to match as best as you can! For example: text-2xl.
```suggestion
      <p className="font-pse-crimson-text scale-50 text-center text- whitespace-nowrap text-white transition-transform duration-300 ease-out sm:scale-60 md:scale-75 lg:scale-100">
        Pi Sigma Epsilon
      </p>
      <p className="font-pse-maitree z-10 -mt-18 scale-50 text-center text- whitespace-nowrap text-white transition-transform duration-300 ease-out sm:-mt-16 sm:scale-60 md:-mt-10 md:scale-75 lg:-mt-6 lg:scale-100">
        University of California, Riverside Chapter
      </p>
```

swedenagu commented on src/components/landing/PSEBanner.tsx:
```diff
@@ -0,0 +1,34 @@
+const PSEBanner = () => {
+  return (
+    <div
+      className={`relative flex h-[309px] w-full flex-col items-center justify-center overflow-clip transition-transform duration-300 ease-out sm:h-[359px] md:h-[409px] lg:h-[459px]`}
```
breakpoints work for the size they're applied at and up. for example, if you want to set a default size for smaller screens and then have medium and larger screens have a larger height for the element, it would look like this:
```suggestion
      className={`relative flex h-1/5 w-full flex-col items-center justify-center overflow-clip transition-transform duration-300 ease-out md:h-1/2`}
```
